Week2/README.md

```diff
@@ -25,6 +25,15 @@ Only watch the below chapters:
 2. Core Programming Syntax 
 3. Variables and Data Types
 
+Read up on the most important topics of next week:
+
+[Variables (var, let, const)](http://javascript.info/variables)
+[Basic Data types (Strings, Numbers, Arrays, Booleans)](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/values.md)
+[Operators](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/operators.md)
+[Special characters and their names](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/names_of_special_characters.md)
+[Naming conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md)
```

Comment:
Hi @gijscor, this doesn't show up as a list when rendered as HTML. Check the link if your Markdown is a bit rusty.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists

Reply:
solved